steps/terraform-precheck.yaml

```diff
@@ -0,0 +1,58 @@
+parameters:
+  - name: keyvaultName
+    default: ''
+
+  - name: keyvaultSecret
+    default: ''
+
+  - name: location
+    default: 'UK South'
+    values:
+      - 'UK South'
+      - 'UK West'
+
+  - name: serviceConnection
+    default: ''
+
+  - name: tfversion
+    default: ''
+
+steps:
+  - checkout: self
+  - checkout: cnp-azuredevops-libraries
```

Comment:
Is this this needed?

Reply:
Yes. Haven't been able to get this working (unable to locate template files/scripts as only the self repo is checked out by default) without checking out the template repo.  Followed [this](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#checkout-path) for reference.